chesster.js

```diff
@@ -1351,7 +1349,10 @@ controller.on('ambient', function(bot, message) {
             return;
         }
         try{
-            var result = spreadsheets.parse_result(message.text, results_options);
+            var result = 
+                spreadsheets.parse_result(
+                    message.text, 
+                    config.gamelinks[channel.name]);
```

Comment:
In this case, parse_result doesn't actually use the options either. So we could just remove it.